data/json/items/gunmod/muzzle.json

```diff
@@ -90,15 +88,14 @@
     "location": "muzzle",
     "mod_targets": [ "pistol", "smg", "rifle" ],
     "install_time": 200,
-    "damage_modifier": -4,
-    "loudness_modifier": -50,
+    "loudness_modifier": -80,
```

Comment:
I did some research into this - the oil filter is actually better in terms of raw decibels. It's just horribly impractical in terms of carrying and shooting the gun.